.circleci/config.yml

```
@@ -0,0 +1,27 @@
+version: 2 
+jobs: 
+  build: 
+    docker: 
+      - image: circleci/golang:1.12
+
+    steps: 
+      - checkout 
+      - restore_cache:
+          keys:
+            - go-mod-v4-{{ checksum "go.sum" }}
+
+      - run:
+          name: Run unit tests
+          command: |
+            go test ./...
```

Slightly better output on circle, gives a nice code coverage report. The `*.out` file extension is already in the `.gitignore`.

```suggestion
            go test ./... -coverprofile coverReport.out; go tool cover -func coverReport.out
```